lib/theme/wallet_theme.dart

```diff
@@ -1,37 +1,33 @@
 import 'package:flutter/material.dart';
-
+import 'extended_theme.dart';
 import 'dark_theme.dart' show darkTheme;
 import 'light_theme.dart' show lightTheme;
+import 'colors.dart';
 
 enum WalletTheme {
   Light,
   Dark,
 }
 
 final walletThemeData = {
-  WalletTheme.Light: lightTheme,
-  WalletTheme.Dark: darkTheme,
+  WalletTheme.Light: lightTheme
+      .copyWith(extensions: <ThemeExtension<dynamic>>[ExtendedTheme.light]),
+  WalletTheme.Dark: darkTheme
+      .copyWith(extensions: <ThemeExtension<dynamic>>[ExtendedTheme.dark]),
 };
 
 Widget witnetLogo(ThemeData theme) {
-  return (theme.brightness == Brightness.light)
-      ? Image(image: AssetImage('assets/img/witnet_logo.png'))
-      : Image(image: AssetImage('assets/img/witnet_logo_light.png'));
-}
-
-class WalletButtonTheme {
-  const WalletButtonTheme({
-    required this.backgroundColor,
-    required this.highlightColor,
-    required this.splashColor,
-    required this.elevation,
-    required this.highlightElevation,
-    required this.shape,
-  });
-  final Color backgroundColor;
-  final Color highlightColor;
-  final Color splashColor;
-  final double elevation;
-  final double highlightElevation;
-  final ShapeBorder shape;
+  return (theme.primaryColor == lightTheme.primaryColor)
```

Comment:
Should we add a dynamic size of the logo? What happen with different window sizes?

Reply:
This is updated in #18